src/martian/compat3.py

```diff
@@ -9,6 +9,6 @@
 
 
 if sys.version_info[0] < 3:
-    str = unicode
+    str = unicode  # NOQA
```

Comment:
Uh? I know this is not part of this PR but usually it is the other way around :)